packages/gatsby/cache-dir/loader.js

```diff
@@ -249,7 +254,8 @@ const queue = {
 
       if (!page) {
         console.log(`A page wasn't found for "${path}"`)
-        return
+        cb()
```

Comment:
Shouldn't this just be `return cb()`?

Reply:
@KyleAMathews ah yes. You're right. I need to update this to be `return cb()`